src/test/java/gift/kakao/LoginTest.java

```diff
@@ -0,0 +1,39 @@
+package gift.kakao;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.net.URI;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.web.client.RestClient;
+
+@SpringBootTest
+public class LoginTest {
+    private final RestClient client = RestClient.builder().build();
```

Comment:
테스트까지 시도하시는 부분 💯 
다만 해당 부분도 빈주입 받으면 될 것 같습니다!
```suggestion
    @Autowired 
    RestClient client;
```